README.md

```diff
@@ -167,28 +167,54 @@ Note that using Lifespan QoS will not have any visual effect.
 
 ## Miscellaneous Options and Windows
 
-### Other Options
+### Participant Configuration
 
-Under the Options->Preferences tab you will find the following menu:
+Under the Options->Participant Configuration tab you will find the following menu:
 
-![Other options](docs/options.png)
+![Other options](docs/participant.png)
 
 You can modify the following settings:
 
-- **Transport Protocol:**  You can select between UDP protocol, TCP LAN Server, TCP WAN Server or TCP Client.
-  With UDP Protocol the applicacion will work sending multicast packets to communicate with other apps.
-  TCP protocol needs a minimal configuration to create the connection between the machines:
+- **Transport Protocol:**  You can select between UDP protocol, TCP LAN Server, TCP WAN Server or TCP Client,
+  and Shared Memory protocol.
+  - **UDP** With UDP Protocol the applicacion will work sending multicast packets to communicate with other apps.
+  - **TCP** TCP protocol needs a minimal configuration to create the connection between the machines:
     - If the applicacion is going to run as a LAN server, it only needs to set the listening port where it is going to
       accept connections (note that firewall must be configured to allow inbound traffic).
     - If the applicacion is going to run as a WAN server, it needs to set the listening port where it is going to
       accept connections and the server WAN address (note that firewall must be configured to allow inbound traffic
       and router must relay listening port traffic to server machine).
     - If the applicacion is going to run as a client it needs to know the IP address of the server (or its WAN address
       if both instances don't share network) and the port where server is listening for connections.
-- **Domain ID:** The user can select different Domain IDs.
+  - **Shared Memory** Activating Shared Memory protocol will use the Shared Memory Transport, a *Fast DDS* feature that
+  allows a faster and more efficientt communication for Participants running in the same host.
+  - **Default** In case no transport has been activated, **Fast DDS** default transports will be used (UDP + SHM).
+
+- **Same host delivery:**  *Fast DDS* has some features that allow Participants running in the same host or process
+  to share resources in order to improve the communication:
+  - **Intraprocess** Allow using Intraprocess delivery when both Endpoints are running in the same process.
+  - **Data Sharing** Allow using Data Sharing delivery when both Endpoints are running in the same host.
+
+- **Domain** The user can select different Domain IDs.
```

Comment:
```suggestion
  - **Shared Memory (SHM):** Activating Shared Memory protocol will use the Shared Memory Transport, a *Fast DDS* feature that
  allows a faster and more efficient communication for Participants running in the same host.
  - **Default:** In case no transport has been activated, **Fast DDS** default transports will be used (UDP + SHM).

- **Same host delivery:**  *Fast DDS* has some features that allow Participants running in the same host or process
  to share resources in order to improve the communication:
  - **Intraprocess:** Allow using Intraprocess delivery when both Endpoints are running in the same process.
  - **Data Sharing:** Allow using Data Sharing delivery when both Endpoints are running in the same host.

- **Domain** The user can select different Domain IDs.
```

You are using `SHM` without having defined what are you referring to.

Reply:
:+1: